sale_test_data_imp/test/sale_order_product_can_be_sold.yml

```diff
@@ -109,7 +109,7 @@
             #~ Validate invoice for this sale order 
             if invoice_id:
                 try:
-                    wf_service = netsvc.LocalService("workflow")
+                    wf_service = netsvc.openerp.workflow
```

Comment:
@KarenKawaii Also you could use this method `invoice_recordset.signal_workflow('signal_name')`. Example [here](https://github.com/odoo/odoo/blob/52484b0132ee3e0063dd67cc3c2eb434f99f5db8/addons/account/wizard/account_invoice_state.py#L22). If you choose use the mention method to call a workflow signals for a record then you will not need to make `import openerp.workflow` and `import openerp.netsvc` in your python files.

I hope this information be useful for you.

Best Regards.